bcgov/mds
# Create new role edit TSF

# Main

- Create new role to edit TSF

# Other
- Creation of EDIT_TSF role in Keycloak 
- Change of MINE_EDIT to EDIT_TSF in PUT & POST method
- Change permission in Core-web for Add button from EDIT_PERMITS to EDIT_TSF
- Change permission in Core-web for Edit button from EDIT_REPORTS to EDIT_TSF
- Update permissions in tests
- Update snapshots
  

# How to test
- Login to Core-web with your IDIR and confirm you cannot Add/Edit a TSF.
- Login to minespace-web with BCEID and confirm that as proponent you can edit a TSF
- Assign the new role EDIT_TSF to your account in keycloak and login to Core-web with your IDIR account.
- Confirm you CAN Add/Edit a TSF

# Notes
https://bcmines.atlassian.net/browse/MDS-4094
